phpunit.xml.dist

```diff
@@ -11,7 +11,7 @@
          failOnWarning="true"
          verbose="true">
 
-    <coverage processUncoveredFiles="true" ignoreDeprecatedCodeUnits="true">
+    <coverage ignoreDeprecatedCodeUnits="true">
```

Comment:
This change is necessary for this bug: https://github.com/sebastianbergmann/phpcov/issues/101
https://github.com/sebastianbergmann/phpcov/issues/101#issuecomment-999974359

Reply:
Very bad news.

Reply:
It seems removing this reduced further the coverage from 84.46% to 84.36%.

To prevent this, I must add phpcov to require-dev instead of the recommended way.

Reply:
If that works, please add it.

If we don't set `processUncoveredFiles="true"`, then files that are not executed when executing tests, i.e. files for which the test is not written at all, will not be included in the coverage calculation, and the coverage will not drop even if test is not written.

Reply:
I am more concerned with the scenario @kenjis described than the tiny coverage drop. I think using Composer instead of the PHAR is a small price to pay. Sorry for causing this by recommending the switch to begin with 😢